### DISCUSSION TITLE
Add support for a second display

### DISCUSSION THREAD

**Discussion by Leowr997** [Softal-io/presentify-community#19] [Ideas]
Hey, anybody knows if support for a second display is coming? As a workaround I’m mirroring, but losing presenter notes at the same time. And these are really handy when teaching a lecture for a whole day 🙂

**Comment by larryebaum** [comment]
Hoping to get this prioritized. I've found not just second display, but multiple. I have a Macbook laptop with 3 external displays and find functionality only works on whatever screen the cursor is on when activating presentify. This includes cursor highlighting.

Further, functionality breaks when using a single display and switch 'desktops'... swiping to another desktop results in no more highlighted cursor (a deal breaker). Other tools like 'Presentation Assistant' handle both multiple desktops and displays without any issue.
